#tranform.yaml
#inputformat yaw pitch roll tx ty tz 
camera1_ypr_t:
  - -33
  - -15
  - -90
  - 0.0376 
  - 0.02177
  - -0.40759
  
camera2_ypr_t:
  - 33
  - -15
  - -90
  - -0.0376
  - 0.02177
  - -0.40759
  
camera3_ypr_t:
  - 30
  - 90 
  - 0
  - -0.18669
  - 0.02998
  - 0.08386

camera4_ypr_t:
  - 30 
  - -180
  - 0
  - -0.01562
  - -0.00934
  - 0.39654

camera5_ypr_t:
  - 30
  - -90
  - 0
  - 0.18705
  - 0.02998
  - 0.11614
  
